src/components/Comments.jsx

```diff
@@ -37,7 +36,7 @@ export function Comments({ reviewId }) {
   return (
     <div className="Comments">
       {comments.map((comment) => {
-        return <Comment {...comment} />;
+        return <Comment {...comment} setComments={setComments} />;
```

Comment:
You're getting 2 errors in the console regarding the missing `key` prop inside where you're mapping. One is here and looks like another is for your category list drop down